src/renderers/index.ts

```diff
@@ -31,7 +32,8 @@ export
 class HTMLWidget extends Widget {
   constructor(html: string) {
     super();
-    this.node.innerHTML = html;
+    let range = document.createRange();
+    this.node.appendChild(range.createContextualFragment(html));
```

Comment:
Probably a topic for the mailing list or outside this PR ...

No mobile support for Jupyter Lab makes sense: the UX isn't really targeting that. But if no mobile / Safari support extends to the various jupyter-js-* `building blocks`, that's a bit more problematic. Limits the reuse of the building blocks in new contexts (e.g., dashboard server)

If we help contribute fixes to  `jupyter-js-*` that make the components compatible with Safari / mobile and they're reasonable implementations (i.e., not 1,000 LOC workarounds), will you accept them?